src/parsec_client/operations/psa_key_attributes.md

```diff
@@ -3,206 +3,191 @@
 The attributes are used to fully describe a cryptographic key: its type, size and what is permitted
 to do with that key.
 
-*Notice:* not all possible attributes are currently supported by Parsec. Please see the [API
+Some of the algorithms defined here are deprecated and should not be used without a valid reason. It
+is at the discretion of the system administrator whether those algorithms are permitted or not.
+
+**Notice:** not all possible attributes are currently supported by Parsec. Please see the [API
 coverage](../../api_coverage.md) for an overview of what Parsec currently supports.
```

Comment:
Do you think it's also worth saying that some of the algorithms/parameters might not be supported by some providers at all, because it's not in their interface? I'm not sure if that's the case with PKCS11 or TPM, but maybe in the future something like this would happen

Reply:
Done! I also squashed all commits together.